test/apiv2/rest_api/test_rest_v1_0_0.py

```diff
@@ -205,7 +207,21 @@ def do_search():
         search.join(timeout=10)
         self.assertFalse(search.is_alive(), "/images/search took too long")
 
-    def validateObjectFields(self, buffer):
+    def test_ping(self):
```

Comment:
Can we add a test to `test/apiv2` as well? Currently, `rest_api` is not firing in the CI and I want to be extra sure we don't regress in between merging and enabling the tests.